other/materials_designer/import_material_from_jarvis_db_entry.ipynb

```diff
@@ -0,0 +1,164 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    jarvis_db_entry = json.dumps(json.loads(df[(df["jid"] == "JVASP-670")].to_json(orient="records"))[0])
</pre><p>Let's isolate the JVASP identifier to a variable</p><p> </p><p>JARVIS_ID = "JVASP-670"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>